packages/example/components/chains/suiStandard/example.tsx

```
@@ -39,6 +39,155 @@
+    console.log('amountBNString', amountBNString);
+
+    const totalBalance = coins.reduce((acc, coin) => {
+      return acc.plus(new BigNumber(coin.balance));
+    }, new BigNumber(0));
+
+    if (totalBalance.lt(amountBN)) {
+      apiFromRef.current?.setValue('result', '余额不足');
+      throw new Error('Insufficient balance');
+    }
+
+    const [primaryCoin, ...mergeCoins] = coins?.filter(
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)